README.md

```diff
@@ -202,40 +238,42 @@ Next from the checkout `/snappy-poc/` directory, build the example
 
 ```
-SPARK_DIST_CLASSPATH=SNAPPY_POC_HOME/assembly/build/libs/snappy-poc-1.0.0-assembly.jar
+SPARK_DIST_CLASSPATH=<snappy_poc_home>/assembly/build/libs/snappy-poc-1.1.1-assembly.jar
 ```
-> Make sure you set the SNAPPY_POC_HOME directory appropriately above
+> Make sure you set the `snappy_poc_home` directory appropriately above
 
-Leave this file open as you will copy/paste the path for SNAPPY_POC_HOME shortly.
+Leave this file open as you will copy/paste the path for `snappy_poc_home` shortly.
 
 Start SnappyData cluster using following command from installation directory. 
 
 ```
 ./sbin/snappy-start-all.sh
 ```
 
-This will start one locator, one server and a lead node. You can understand the roles of these nodes [here](https://github.com/SnappyDataInc/snappydata/blob/master/docs/GettingStarted.md#snappydata-cluster-explanation)
-
-
+This will start one locator, one server and a lead node. You can understand the roles of these nodes [here](https://snappydatainc.github.io/snappydata/architecture/cluster_architecture/)
 
 Submit the streaming job to the cluster and start it (consume the stream, aggregate and store).
 > Make sure you copy/paste the SNAPPY_POC_HOME path from above in the command below where indicated
 
 ```
-./bin/snappy-job.sh submit --lead localhost:8090 --app-name AdAnalytics --class io.snappydata.adanalytics.SnappySQLLogAggregatorJob --app-jar SNAPPY_POC_HOME/assembly/build/libs/snappy-poc-1.0.0-assembly.jar --stream
+./bin/snappy-job.sh submit --lead localhost:8090 --app-name AdAnalytics --class io.snappydata.adanalytics.SnappyLogAggregator --app-jar $SNAPPY_POC_HOME/assembly/build/libs/snappy-poc-1.1.1-assembly.jar
 ```
 
-SnappyData supports "Managed Spark Drivers" by running these in Lead nodes. So, if the driver were to fail, it can automatically re-start on a standby node. While the Lead node starts the streaming job, the actual work of parallel processing from kafka, etc is done in the SnappyData servers. Servers execute Spark Executors collocated with the data. 
+SnappyData supports "Managed Spark Drivers" by running these in Lead nodes. So, if the driver were to fail, it can
+automatically re-start on a standby node. While the Lead node starts the streaming job, the actual work of parallel
+processing from kafka, etc is done in the SnappyData servers. Servers execute Spark Executors collocated with the data. 
 
 Start generating and publishing logs to Kafka from the `/snappy-poc/` folder
 ```
 ./gradlew generateAdImpressions
 ```
 
-You can see the Spark streaming processing batches of data once every second in the [Spark console](http://localhost:4040/streaming/). It is important that our stream processing keeps up with the input rate. So, we note that the 'Scheduling Delay' doesn't keep increasing and 'Processing time' remains less than a second.
+You can monitor the streaming query processing on the [Structured Streaming UI](http://localhost:5050/structuredstreaming/). It is
```

Comment:
Should the link title be "SnappyData Structured Streaming UI" as it's something we have implemented?